internal/scaffold/ansible/watches.go

```diff
@@ -28,6 +36,12 @@ type Watches struct {
 	Resource         scaffold.Resource
 }
 
+type watchesYAML struct {
+	GroupVersionKind schema.GroupVersionKind `yaml:",inline"`
+	Role             string                  `yaml:"role,omitempty"`
+	Playbook         string                  `yaml:"playbook,omitempty"`
+}
+
```

Comment:
The yamlWatches was removed above, so what it was added here?
I think we should keep the previous yamlWatches instead of.  
@joelanford are you ok with the WATCH changes made for Helm?